traversal/src/main/scala/overflowdb/traversal/Traversal.scala

```diff
@@ -202,9 +207,41 @@ class Traversal[A](elements: IterableOnce[A])
       }.getOrElse(Traversal.empty)
     }
 
+  /** aggregate all objects at this point into the given collection (side effect)
+   * e.g. `mutable.ArrayBuffer.empty[A]` */
+  def aggregate(into: mutable.Growable[A]): Traversal[A] =
+    sideEffect(into.addOne(_))
+
+  /** sort elements by their natural order */
+  @Doc("sort elements by their natural order")
+  def sorted(implicit ord: Ordering[A]): Traversal[A] =
+    Traversal(elements.to(Seq).sorted)
+
+  /** sort elements by the value of the given transformation function */
+  @Doc("sort elements by the value of the given transformation function")
+  def sortBy[B](f: A => B)(implicit ord: Ordering[B]): Traversal[A] =
+    Traversal(elements.to(Seq).sortBy(f))
+
   def enablePathTracking: PathAwareTraversal[A] =
     PathAwareTraversal.from(elements)
 
+  /** group elements and count how often they appear */
+  @Doc("group elements and count how often they appear")
```

Comment:
is this a common pattern? I don't see how one cannot use the expression directly themselves given that `groupCount` is public. sometimes less is more

Reply:
I don't understand "use the expression directly themselves" - can you elaborate please? do you mean just having `groupCount(A=>B)` and let the user figure out to pass `identity`?

Reply:
yeap

Reply:
in that case i'd like to keep the `.groupCount` without args, yes it can be quite useful
http://tinkerpop.apache.org/docs/current/reference/#groupcount-step